packages/mdc-top-app-bar/README.md

```diff
@@ -120,11 +140,7 @@ Fixed top app bars stay at the top of the page and elevate above the content whe
       <a href="#" class="material-icons mdc-top-app-bar__navigation-icon">menu</a>
       <span class="mdc-top-app-bar__title">Title</span>
     </section>
-    <section class="mdc-top-app-bar__section mdc-top-app-bar__section--align-end" role="top-app-bar">
-      <a href="#" class='material-icons mdc-top-app-bar__action-item' aria-label='Download'>file_download</a>
-      <a href="#" class='material-icons mdc-top-app-bar__action-item' aria-label='Print this page'>print</a>
-      <a href="#" class='material-icons mdc-top-app-bar__action-item' aria-label='Bookmark this page'>bookmark</a>
-    </section>
+    <!-- Action Icons --> 
```

Comment:
I'd just replace these with `...`

Actually if the top-level element is the only thing affected we could potentially even just `...` the entire contents beyond the first Short example.